src/app/helptext/services/components/service-netdata.ts

```diff
@@ -0,0 +1,75 @@
+import { T } from '../../../translate-marker';
+import { Validators } from '@angular/forms';
+import {
+    regexValidator
+  } from '../../../pages/common/entity/entity-form/validators/regex-validation';
+
+export default {
+    global_paratext: T('Global Settings'),
+
+    history: {
+        placeholder: T('History'),
+        tooltip: T('The number of <samp>entries</samp> the netdata daemon will\
```

Comment:
Why <samp> here?  Is "entries" a literal term?  The sentence is also a little tortured:
```suggestion
        tooltip: T('The number of entries the netdata daemon keeps\
```